win32ss/gdi/ntgdi/freetype.c

```diff
@@ -3227,12 +3277,21 @@ ftGdiGetGlyphOutline(
     IntLockFreeType();
 
     /* Scaling transform */
-    /*if (aveWidth)*/
+    if (widthRatio != 1.0)
     {
+        FT_Matrix scaleMat;
+        scaleMat.xx = FT_FixedFromFloat(widthRatio);
+        scaleMat.xy = 0;
+        scaleMat.yx = 0;
+        scaleMat.yy = FT_FixedFromFloat(1.0);
+
+        FT_Matrix_Multiply(&scaleMat, &transMat);
+        needsTransform = TRUE;
+    }
 
+    {
```

Comment:
Can you add a comment about what this code block does, compared to the "widthratio != 1.0" block from above?

Reply:
OK. Added.